DfE.FindInformationAcademiesTrusts.Data.AcademiesDb/Utilities.cs

```diff
@@ -0,0 +1,20 @@
+namespace DfE.FindInformationAcademiesTrusts.Data.AcademiesDb;
+
+public interface IUtilities
```

Comment:
Currently this only works with addresses and is string manipulation my proposal is 
StringFormattingUtilities.BuildAddressString(...)

Or extend StringExtensions to be more generic but atleast still scoped to string extensions/manipulation

Reply:
Have renamed to `StringFormattingUtilities`